docs/how-to/jupyter-notebook.rst

```diff
@@ -83,6 +83,52 @@ Confirm the GPU is detected and usable by running:
 
    tf.config.list_physical_devices('GPU')
 
+Create an Intel GPU-enabled notebook
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can create an Intel GPU-enabled Jupyter Notebook with IPEX or ITEX.
+
+.. note::
+
+   To launch an Intel GPU-enabled notebook, you must first :doc:`Enable Intel GPUs <../how-to/enable-gpus/enable-intel-gpu>`.
```

Comment:
Let's use `:ref:` + the tag here for the internal linking. I committed this change but probably you didn't refresh your local PR with my change and got overwritten.

Reply:
done